packages/collector/test/tracing/logger/log4js/test.js

```diff
@@ -38,23 +44,22 @@ mochaSuiteFn('tracing/logger/log4js', function () {
       suffix = '(using level method)';
     }
 
-    it(`must not trace info ${suffix}`, () =>
-      trigger('info', 'Info message - must not be traced.', useLogMethod).then(() =>
-        testUtils.retry(() =>
-          agentControls.getSpans().then(spans => {
-            const entrySpan = testUtils.expectAtLeastOneMatching(spans, [
-              span => expect(span.n).to.equal('node.http.server'),
-              span => expect(span.f.e).to.equal(String(controls.getPid())),
-              span => expect(span.f.h).to.equal('agent-stub-uuid')
-            ]);
-            testUtils.expectAtLeastOneMatching(spans, span => {
-              checkNextExitSpan(span, entrySpan);
-            });
-            const log4jsSpans = testUtils.getSpansByName(spans, 'log.log4js');
-            expect(log4jsSpans).to.be.empty;
-          })
-        )
-      ));
+    it(`must not trace info ${suffix}`, async () => {
+      await trigger({ level: 'info', message: 'Info message - must not be traced.', useLogMethod });
+      return retry(async () => {
+        const spans = await agentControls.getSpans();
+        const entrySpan = expectAtLeastOneMatching(spans, [
+          span => expect(span.n).to.equal('node.http.server'),
+          span => expect(span.f.e).to.equal(String(controls.getPid())),
+          span => expect(span.f.h).to.equal('agent-stub-uuid')
+        ]);
+        expectAtLeastOneMatching(spans, span => {
+          checkNextExitSpan(span, entrySpan);
+        });
+        const log4jsSpans = getSpansByName(spans, 'log.log4js');
+        expect(log4jsSpans).to.be.empty;
+      });
+    });
```

Comment:
Mostly refactored some `.then` to async/await style in the test, that's why the diff is a bit larger than absolutely necessary for the actual fix.